### PR TITLE
Fw updates

### PR DESCRIPTION
**Pull request by readerror67**
Just ran astyle.

### CODE REVIEW

File: src/fw_nav.c [hunk 1]
```diff
@@ -40,6 +43,19 @@ void fw_nav_reset(void)
     }
 }
 
+void fw_FlyTo(void)  // PatrikE CruiseMode version
+{
+#define GEO_SKALEFACT    89.832f  // Scale to match meters
```
Comment (trollcop) [inline src/fw_nav.c]: move this shit outside the function

File: src/fw_nav.c [hunk 2]
```diff
@@ -40,6 +43,19 @@ void fw_nav_reset(void)
     }
 }
 
+void fw_FlyTo(void)  // PatrikE CruiseMode version
+{
+#define GEO_SKALEFACT    89.832f  // Scale to match meters  
+    int32_t holdHeading = GPS_ground_course / 10;
+    if (holdHeading > 180)
+        holdHeading -= 360;
+    float scaler = ( GEO_SKALEFACT / GPS_scaleLonDown) * cfg.fw_cruise_distance;
```
Comment (trollcop) [inline src/fw_nav.c]: spacing, and move vars declarations to top

File: src/mixer.c [hunk 3]
```diff
@@ -449,6 +449,29 @@ void writeServos(void)
...
+            motor[1] = rcCommand[THROTTLE];
+        }
+
+        if (!f.ARMED || ((rcData[THROTTLE]) < mcfg.mincheck && feature(FEATURE_MOTOR_STOP))) {
+            motor[0] = mcfg.mincommand;
+            motor[1] = mcfg.mincommand;
+        }
+    }
+
     uint8_t i;
```
Comment (trollcop) [inline src/mixer.c]: this needs to be at top of function

File: src/mw.c [hunk 4]
```diff
@@ -109,17 +109,39 @@ void annexCode(void)
...
+                } else {
+                    prop2 -=  cfg.dynThrPID;
+                }
+            }
         }
+        // APA dynamic PID adjustemnt, depending on Angle of attack
+        uint16_t MaxBrkpoint = 300; // Max angle
+        if (angle[1] > 20)
+            prop2 -= ((uint16_t)cfg.dynThrPID * (min(angle[1], MaxBrkpoint)) >> 8);
+        prop2 = max((128 - cfg.dynThrPID), prop2);
```
Comment (trollcop) [inline src/mw.c]: this whole thing above makes no fucking sense

**Comment by readerror67** [comment]
rage closing.

**Comment by trollcop** [comment]
Gotta talk to patrick to explain what that dynthrottle changes are, they
look too copypasted and I never understood it well

**Pull request closed without merging**